docs/parity-status.md

```diff
@@ -1,77 +1,31 @@
 ---
 id: parity-status
-title: Component and Module API Parity Status
+title: API Parity
 ---
 
-This doc reports the status progress of core components modules/APIs integration for vnext. The high priority components and modules tracked here will be in alignment with the [React Native Lean Core](https://github.com/facebook/react-native/issues/23313) effort, others not in Lean Core we plan on shipping as separate packages in [future milestones](https://github.com/microsoft/react-native-windows/milestones).
+All [React Native Components and APIs](https://reactnative.dev/docs/components-and-apis) that are a part of the [React Native Lean Core](https://github.com/facebook/react-native/issues/23313) effort are now all supported in React Native for Windows.
 
-If there is a discrepancy here, or features are not being tracked that should be, please [submit an issue under vnext](https://github.com/microsoft/react-native-windows/issues/new?labels=vnext&template=vnext.md) to let us know.
+For a more closely monitored look at our work in progress, check out the:
+- [API Completion](https://github.com/microsoft/react-native-windows/labels/API%20Completion) label on issues
+- [GitHub project board - Core Features](https://github.com/microsoft/react-native-windows/projects/7)
+- [GitHub project board - Community Module Support](https://github.com/microsoft/react-native-windows/projects/23)
 
-You can keep track of our component and module API parity more closely on our [GitHub project borad](https://github.com/microsoft/react-native-windows/projects/7).
+> Please feel free to leave comments on give us a 👍 on issues to help us prioritize better!
 
-### Components

```

Comment:
So if not here, where and how do we want to capture this information. We have issues tracked for some, but I don't know if that's the most discoverable way to get this information. I had been imagining this information grew and was updated (as well as dumping all the `current` info). See [this issue](https://github.com/microsoft/react-native-windows/issues/3587) for more discussion on this matter.

Reply:
@chrisglein We chose to eliminate the high level tables since there are only few props/methods deltas at this time and the tables are always looking staler than what is supported - it is not providing useful information to customers since things that are missing from the table are assumed to be missing from Windows implementation. The right place for documenting differences will be in the API docs themselves which @kikisaints is working on a plan to add Windows deltas. Until then, communicating that all types are supported and for missing props/methods - redirecting customers to the issues feels like a good thing to do.